src/mmw/js/src/modeling/models.js

```diff
@@ -276,10 +275,8 @@ var ScenarioModel = Backbone.Model.extend({
         this.set('inputs', new ModificationsCollection(attrs.inputs));
         this.set('modifications', new ModificationsCollection(attrs.modifications));
 
-        this.on('change:project change:name change:census', this.attemptSave, this);
-        this.get('inputs').on('add', this.attemptSave, this);
+        this.on('change:project change:name', this.attemptSave, this);
         this.get('modifications').on('add remove change', this.updateModificationHash, this);
-        this.get('modifications').on('add remove', this.attemptSave, this);
```

Comment:
So we no longer save at the time the modification is made. That's find presuming short turn around for processing. But we might need to be hanging out on the order of minutes and if we don't save until the very end, the chance for data loss increases significantly.